cmd/podman/system/varlink.go

```diff
@@ -16,11 +16,12 @@ var (
   Tools speaking varlink protocol can remotely manage pods, containers and images.
 `
 	varlinkCmd = &cobra.Command{
-		Use:   "varlink [flags] [URI]",
-		Args:  cobra.MinimumNArgs(1),
-		Short: "Run varlink interface",
-		Long:  varlinkDescription,
-		RunE:  varlinkE,
+		Use:        "varlink [flags] [URI]",
+		Args:       cobra.MinimumNArgs(1),
+		Short:      "Run varlink interface",
+		Long:       varlinkDescription,
+		RunE:       varlinkE,
+		Deprecated: "Please see `podman system service' for RESTful APIs",
```

Comment:
```suggestion
		Deprecated: "Please see 'podman system service' for RESTful APIs",
```
different single quotes, I removed the back tick one in front, you could replace the ending one if you prefer as long as they match.